app/views/works/index.html.erb

```diff
@@ -0,0 +1,108 @@
+<article>
+  <h1>List of Works</h1>
+</article>
+
+<article>
+  <h2>Albums</h2>
+  <section>
+    <table class="table">
+      <thead>
+        <tr>
+          <th class = "table_grid">Votes</th>
+          <th class = "table_grid">Title</th>
+          <th class = "table_grid">Created By</th>
+          <th class = "table_grid">Published</th>
+          <th class = "table_grid">Upvote</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @works.each do |work| %>
+          <% if work.category == "album" %>
```

Comment:
This view looks great! Thought experiment: These three article tags feel pretty repeated, with the exception of the header and what the `work.category` is... if given more time, how could we refactor this?